### PR TITLE
Productmaster

### PR DESCRIPTION
**Pull request by pathompongbo**
add and update more fields.

### CODE REVIEW

File: productmaster/productmaster.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  portion:
+    type: string
+ stock_qty:
+    type: int
+ status:
+    type: string
+ preorder:
+    type: string
+ preorder_fix_date:
+    type: int
```
Comment (pinnareet) [inline productmaster/productmaster.yaml:124]: integer ไม่ใช่ int จ้า ฝากแก้ด้วยจ้า

**Review by pinnareet — CHANGES_REQUESTED** [review]

File: productmaster/productmaster.yaml [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  pr_puqty:
+    type: string
+  pr_sa_method:
+    type: string
+  pr_sucode1:
+    type: string
+  pr_suref3:
+    type: string
+  prtype:
+    type: string
```
Comment (pinnareet) [inline productmaster/productmaster.yaml:98]: prtype should be enum (I, C)

**Comment by pinnareet** [comment]
more details here https://docs.google.com/spreadsheets/d/16NUZn-8vH8kW4bWyAUvPyn-0fYfWAve_2gNvXEk_AM4/edit?usp=sharing

**Comment by pinnareet** [comment]
need to update more stuff but will pull for now per mickey's request

**Comment by thanakijwanavit** [comment]
hello, are we ready for merge?

**Review by pinnareet — APPROVED** [review]

**Pull request merged**